src/main/java/com/kichik/pecoff4j/ModifiableStructure.java

```diff
@@ -0,0 +1,21 @@
+package com.kichik.pecoff4j;
+
+import com.kichik.pecoff4j.io.IDataWriter;
+
+import java.io.IOException;
+
+public interface ModifiableStructure {
```

Comment:
What about the interface makes it modifiable? I feel like I'm missing something with the naming of this.

Reply:
Good point, all of the structure classes are modifiable already now. I would propose to rename it to RebuildableStructure, so it better reflects its main method (rebuild).